wrappers/src/shared_realm_cs.cpp

```diff
@@ -46,6 +46,39 @@ namespace binding {
     }
 }
     
+    template<typename KeyType>
+    Object* create_object_unique(SharedRealm* realm, Table* table, KeyType key, bool try_update, std::function<size_t (size_t)> find_first, std::function<void (size_t, size_t)> set_unique, std::string key_to_string, bool& is_new)
+    {
+        (*realm)->verify_in_write();
+        const std::string object_name(ObjectStore::object_type_for_table_name(table->get_name()));
+        auto& object_schema = *realm->get()->schema().find(object_name);
```

Comment:
I'm not sure how we could get in a situation where the schema doesn't contain the object name.